apps/atomy-q/API/app/Services/QuoteIntake/NormalizationOverrideService.php

```
@@ -0,0 +1,368 @@
+{
+    public function __construct(
+        private QuoteSubmissionReadinessService $readiness,
+        private DecisionTrailRecorder $decisionTrail,
+    ) {}
+
+    /**
+     * @param array{override_data: array<string, mixed>, reason_code: string, note?: string|null} $validated
+     * @param string $actorUserId
+     * @return array{line: NormalizationSourceLine, readiness: array{has_blocking_issues: bool, blocking_issue_count: int, next_status: string}}
+     */
+    public function createSourceLine(QuoteSubmission $submission, string $actorUserId, array $validated): array
```

PHPMD (2.15.0)

[warning] Avoid unused parameters such as '$actorUserId'. (undefined)

(UnusedFormalParameter)